space2stats_api/src/space2stats/app.py

```diff
@@ -0,0 +1,54 @@
+from contextlib import asynccontextmanager
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+
+from typing import Any, Dict, List
+
+from starlette.requests import Request
+
+from .db import connect_to_db, close_db_connection
+from .main import get_summaries_from_geom, get_available_fields, SummaryRequest
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    """FastAPI Lifespan."""
+    # Create Connection Pool
+    await connect_to_db(app)
+    yield
+    # Close the Connection Pool
+    await close_db_connection(app)
```

Comment:
proper ways to init/close the database pool when using the application

Reply:
thank you!